cli/cmd/volume/acivolume.go

```diff
@@ -30,27 +34,27 @@ import (
 	"github.com/docker/compose-cli/progress"
 )
 
-// ACICommand manage volumes
-func ACICommand() *cobra.Command {
+// Command manage volumes
+func Command(ctype string) *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "volume",
 		Short: "Manages volumes",
 	}
 
 	cmd.AddCommand(
-		createVolume(),
+		createVolume(ctype),
 		listVolume(),
 		rmVolume(),
 		inspectVolume(),
 	)
 	return cmd
 }
 
-func createVolume() *cobra.Command {
-	aciOpts := aci.VolumeCreateOptions{}
+func createVolume(ctype string) *cobra.Command {
+	var opts interface{}
 	cmd := &cobra.Command{
-		Use:   "create --storage-account ACCOUNT VOLUME",
-		Short: "Creates an Azure file share to use as ACI volume.",
+		Use:   "create [OPTIONS] VOLUME",
```

Comment:
Could we have a different help message based on the context?

Reply:
Wasn't the intent to have a uniform UX ?
Otherwise I don't see the benefit to have shared code and make each and every line context-conditional :P

Reply:
We want... both!

The UX should be as similar as possible but we want to help users do the right thing where we can. For ACI, we need them to specify the storage account on volume creation so hinting at that in the help would help users.

The failure case here isn't so bad; the user will not specify it and get an error saying please specify it.